src/main/java/com/github/games647/flexiblelogin/FlexibleLogin.java

```diff
@@ -131,12 +131,12 @@ public void onInit(GameInitializationEvent initEvent) {
     }
 
     private void registerCommands() {
-        commandManager.register(this, injector.getInstance(LoginCommand.class).buildSpec(), "login", "log");
+        commandManager.register(this, injector.getInstance(LoginCommand.class).buildSpec(), "login", "log", "l");
```

Comment:
This command already existed before, but it was removed because of this:
https://forums.spongepowered.org/t/flexiblelogin-v0-17-for-sponge-7-auth-plugin-2fa/8872/233

Reply:
What do you think about option in configuration? That alias is very much known by players on AuthMeReloaded-powered servers and they will can't log in (players usually don't read documentations).

Reply:
Yes a configuration is perfect.